internal/controller/operatorconfigmap_controller.go

```diff
@@ -462,6 +474,13 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI(storageClients *v1al
 			return fmt.Errorf("failed to own csi rbd driver: %v", err)
 		}
 		rbdDriver.Spec.GenerateOMapInfo = ptr.To(c.shouldGenerateRBDOmapInfo())
+		if len(topologyKey) > 0 {
+			rbdDriver.Spec.NodePlugin = &csiopv1a1.NodePluginSpec{
+				Topology: &csiopv1a1.TopologySpec{
+					DomainLabels: topologyKey,
+				},
+			}
+		}
```

Comment:
By design we leave the specific drivers CRs to be owned and used for user realm. Please apply all of your changes to the driver template part of the csiOperatorConfig